change AHT1X support from initial to full

device111 opened the pull request:
- both sensor address where supported
- dual operation support
- timing optimized
- supported commands added

## Description:

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [x] The pull request is done against the latest dev branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR.
  - [x] The code change is tested and works on core 2.6.1
  - [x] The code change pass travis tests. **Your PR cannot be merged unless tests pass**
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

device111 committed:
change AHT1X support from initial to full

- both sensor address where supported
- dual operation support
- timing optimized
- supported commands added

arendst commented:
Pls revisit the code changes I made to integrate it in the tasmota i2c environment

Fix the use of global temperature and humidity

Fix the use of system provided celsius/fahrenheit

Remove unneeded warnings valid for all i2c drivers when more than one is in the compile set as is the case in tasmota-sensors.

device111 committed:
Fix some issues

- fix global temp and humidity
- fix unneeded defines

device111 committed:
Delete tasmota.ino.cpp

device111 committed:
comment corrected

arendst commented:
Pls revisit the code changes I made to integrate it in the tasmota i2c environment and supporting both global temperature / humidity and celsius/fahrenheit

device111 committed:
change the use of global temp and hum

arendst commented on tasmota/xsns_63_aht1x.ino:
```diff
@@ -22,137 +22,160 @@
+     {
+       float aht_temp = ConvertTemp(tem);
+       float aht_hum = ConvertPressure(hum);
+
```
Almost there but ConvertPressure should be ConvertHumidity.

Why don't you do it during AHT1XRead as I did. That way global info is updated every 2 seconds independant of when data is presented.

device111 replied:
ok, i rechange code complete to read the sensor every Second and only show the values if there are needed.
Thanks!

device111 committed:
Fixes

- change Global Humidity and Temperature to Read-Sensor Section
- fix wrong Pressure Call

device111 commented:
uuups...sorry. Shame on me...I've don't have seen the pressure function. I'm blinded...:-)
